composer.json

```diff
@@ -29,10 +28,10 @@
     "symfony/dependency-injection": "^3.4||^4.3",
     "symfony/process": "^2.1||^4.1",
     "symfony/serializer": "^3.3",
-    "symfony/yaml": "^3.3||^4.0"
-  },
-  "suggest": {
-    "magento/magento-cloud-docker": "^1.0"
+    "symfony/yaml": "^3.3||^4.0",
+    "magento/magento-cloud-components": "^1.0.1",
+    "magento/magento-cloud-docker": "^1.0.0",
+    "magento/magento-cloud-patches": "^1.0.0"
```

Comment:
Ideally we should keep these alphabetized, but that's minor

Reply:
Done